ExampleProject/platforms/ios/ExampleProject/Plugins/clevertap-cordova/CleverTapPlugin.m

```diff
@@ -429,6 +431,26 @@ - (void)deleteNotificationChannelGroup:(CDVInvokedUrlCommand *)command {
     NSLog(@"deleteNotificationChannelGroup is no-op in iOS");
 }
 
+
+#pragma mark - Push Notification Delegate
+
+- (void)pushNotificationTappedWithCustomExtras:(NSDictionary *)customExtras {
+    
+    NSMutableDictionary *jsonDict = [NSMutableDictionary new];
+    
+    if (customExtras != nil) {
+        jsonDict[@"customExtras"] = customExtras;
+    }
+    
+    NSString *jsonString = [self _dictToJson:jsonDict];
+    
```

Comment:
Please verify how custom extras are returned in Android platform. Are they adding Push Notification KV to **customExtras** key?

Reply:
Same is being done in Android, no change required from iOS then